.jenkins/md5.txt

```diff
@@ -0,0 +1,13 @@
+cf20eac67f2a214d757762b11bd34632  atmos_8xdaily.tile1.nc
```

Comment:
This filename doesn't explain what configuration was run, or what machine it was made on, and it adds an extra place where we're storing md5 files. Instead of using this md5.txt file, can you directly use the one included in `tests/pytest/reference`? If you need a different version of the files for different machines, you can make a `tests/pytest/reference/<machine name>` directory, similiar to `tests/pytest/reference/circleci`.